lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -30,6 +30,7 @@ class RequestHandler {
 
   virtual void operator()(const llvm::json::Object &request) = 0;
```

Comment:
Should we also mark this `const`? I don't think any of the request handlers are using any state at the moment.

Reply:
Yup, I was going to it in a separate commit, but I can include it here.